.github/workflows/add-dependabot-pr-to-project.yml

```diff
@@ -0,0 +1,17 @@
+name: Add Dependabot PR's to project
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  add-to-project:
+    name: Add dependabot pr to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
+        with:
+          project-url: https://github.com/orgs/<orgName>/projects/<projectNumber>
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
+          labeled: Dependency Upgrade
```

Comment:
Just a hunch, but shouldn't this be the full label name? In that case, it would be "Type: Dependency Upgrade".
Not sure if it works that way, though.

Reply:
The semicolon is probably an issue there. If this does not work we should change the label and dependabot.yml

Reply:
Not sure if single quotes might fix that? That's at least a solution sometimes with yaml.